gravitee-apim-console-webui/src/management/api/entrypoints-v4/edit/api-entrypoints-v4-edit.component.html

```diff
@@ -23,7 +23,30 @@
   <form [formGroup]="form" *ngIf="form; else loader" (ngSubmit)="onSaveEntrypointConfig()">
     <div *ngIf="entrypointSchema" class="api-creation-v4__form-container">
       <h3>{{ entrypointName }}</h3>
-      <gio-form-json-schema [jsonSchema]="entrypointSchema" [formControlName]="entrypoint.type"></gio-form-json-schema>
+
+      <mat-card class="qos">
+        <gio-banner-info>
```

Comment:
Suggestion: Maybe we can create a component for this banner in order to avoid duplicating it in 2 components ?

Reply:
You're right, I hesitated to create a dedicated component to handle the whole Qos thing. But I found it came with more complexity than with code duplication. And it's only use at these 2 places.
I can probably rework a little bit the PR 😅